metis-indexing/src/main/java/eu/europeana/indexing/mongo/property/MongoPropertyUpdaterImpl.java

```diff
@@ -59,8 +59,7 @@ class MongoPropertyUpdaterImpl<T> implements MongoPropertyUpdater<T> {
     this.current = current;
     this.updated = updated;
     this.mongoServer = mongoServer;
-    this.updateOperators = Optional.ofNullable(updateOperators).stream().flatMap(Collection::stream)
-                                   .collect(Collectors.toList());
+    this.updateOperators = Optional.ofNullable(updateOperators).stream().flatMap(Collection::stream).collect(Collectors.toList());
```

Comment:
`this.updateOperators = Optional.ofNullable(updateOperators).stream().flatMap(Collection::stream).collect(Collectors.toList());`
to
` this.updateOperators = Optional.ofNullable(updateOperators).stream().flatMap(Collection::stream).toList();`

Reply:
No. This is a mutable list and should remain so.
Changing it will mean that code will start throwing exceptions...